src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -268,11 +268,9 @@ def get(self, request: Request, project: Project) -> Response:
                 should_get_coverage = codecov_enabled(project.organization, request.user)
                 scope.set_tag("codecov.enabled", should_get_coverage)
                 if should_get_coverage:
-                    codecov_data, err = fetch_codecov_data(config=current_config)
+                    codecov_data = fetch_codecov_data(config=current_config)
```

Comment:
We will report the error within `fetch_codecov_data` making the code clearer and typing stricter.